_posts/2020-09-20-Linux-Install-Script.md

```diff
@@ -6,11 +6,9 @@ author: "niebert"
 lang: "en"
 ---
 
-If you plan to be installing Jamulus on many Linux machines, you may want to try this script.
+IThanks to niebert, if you plan to install Jamulus on many Linux machines, you can try this script.
```

Comment:
Looks like a typo here.

Reply:
I already solved that.

Reply:
ok

Reply:
You should be able to review it again (with a new GitHub review)

Reply:
I was doing it :)